posthog/api/authentication.py

```diff
@@ -118,11 +117,14 @@ def create(self, validated_data: Dict[str, str]) -> Any:
         if not user:
             raise serializers.ValidationError("Invalid email or password.", code="invalid_credentials")
 
-        require_verification_feature = posthoganalytics.feature_enabled(
-            "require-email-verification", str(user.distinct_id)
+        require_verification_feature = (
+            posthoganalytics.get_feature_flag(
+                "require-email-verification", str(user.distinct_id), person_properties={"email": user.email}
```

Comment:
@neilkakkar if I don't include the user email address in here, and we set the feature flag to be true for only certain email addresses, it doesn't always evaluate properly. Is that expected?